### DISCUSSION TITLE
Custom active bullet

### DISCUSSION THREAD

**Discussion by Ravvsky** [nolimits4web/swiper#5526] [Q&A]
Hi,

I want to make my active bullet a div with a text and inactive bullets to be jest regular bullets.

I tried this but it doesn't work
``
<Swiper
        modules={[Pagination, A11y]}
        loop={true}
        slidesPerView={1}
        pagination={{ clickable: true, el: ".pagi" ,
        renderBullet: function(index, className) { 
          let name = "text";
        
          return `<div class="${className}">${className.includes('active') ? name : ''}  </div>`}
                  
      }}
      >

**Comment by Arhell** [comment]
Hi, if it doesn't work out, try using css to do what you want.
